core-bundle/src/Resources/contao/models/PageModel.php

```diff
@@ -1374,7 +1390,22 @@ public function getAbsoluteUrl($strParams=null)
 		$this->loadDetails();
 
 		$objRouter = System::getContainer()->get('router');
-		$strUrl = $objRouter->generate(RouteObjectInterface::OBJECT_BASED_ROUTE_NAME, array(RouteObjectInterface::CONTENT_OBJECT => $this, 'parameters' => $strParams), UrlGeneratorInterface::ABSOLUTE_URL);
+
```

Comment:
The docbloc currently says `@throws RouteNotFoundException` - this would be a breaking change because now we're throwing a `ResourceNotFoundExeception` instead. Or the docbloc is wrong :stuck_out_tongue:. 

(same for `getPreviewUrl()`)

Reply:
Fixed in 9c336f12caa4200814403bbfcaed70b9229ce965.